src/main/java/org/broadinstitute/hellbender/tools/copynumber/CondenseReadCounts.java

```diff
@@ -0,0 +1,203 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.util.BlockCompressedOutputStream;
+import htsjdk.samtools.util.IOUtil;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.SimpleCountCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.SimpleCount;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@CommandLineProgramProperties(summary = "condenses read-counts in .tsv format into a new file also in .tsv format",
+        oneLineSummary = "condenses read-counts in .tsv format into a new file also in .tsv formas" ,
+        programGroup = CopyNumberProgramGroup.class)
```

Comment:
If this needs to be merged quickly, we should add the `@ExperimentalFeature`.  This tool doesn't have documentation, lacks tests, breaks the design of the CNV collection classes, and is otherwise idiosyncratic.  It is also written primarily to satisfy a specific use case in a "downstream" pipeline.  We can go back and address the former, but should also be sure the latter is justified (@mwalker174 recall that we just discussed a related issue).  That said, I do think a tool that combines counts will be useful, so we should take care with the final design.

Reply:
formas -> format